docs/user-guide/integrations/Nextcloud/README.md

```diff
@@ -41,13 +41,13 @@ Enter the URL of your OpenProject instance and your access token which you can f
 
 **Display of OpenProject in the Nextcloud dashboard**
 
-On the Nextcloud dashboard you can add an OpenProject widget. Display the latest changes to your project's work packages to keep an eye on your ongoing project activites directly from your Nextcloud instance.
+On the Nextcloud dashboard you can add an OpenProject widget. Display the latest changes to your project's work packages to keep an eye on your ongoing project activities directly from your Nextcloud instance.
 
-![Add_OpenPorject_widget](../faq/Add_OpenPorject_widget.png)
+![Add_OpenProject_widget](../faq/Add_OpenPorject_widget.png)
```

Comment:
I did not change the file name. There is https://github.com/opf/openproject/blob/dev/docs/user-guide/integrations/Integrations-faq/Add_OpenPorject_widget.png - maybe that is where this file lives now?
There is no `faq` directory any more.

Reply:
See #9370